extension/ts/background-scripts/background.js/telemetry/metrics.yaml

```diff
@@ -278,3 +278,21 @@ errors:
     notification_emails:
       - glean-team@mozilla.com
     expires: never
+
+test:
```

Comment:
I wouldn't mix test stuff here. We should write a test to call bergamot translator and check that telemetry values are set using getTestValue methods

Reply:
Sure thing, nuke it before or after merge, your PR your choice :)